cibyl/config.py

```diff
@@ -0,0 +1,36 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+
+LOG = logging.getLogger(__name__)
+
+
+class Config:
+    """.
+    """
+
+    def __init__(self, path=None):
```

Comment:
On my proposal for this class: https://github.com/rhos-infra/cibyl/pull/19, the 'path' argument is on the 'load' method instead. I did not see any use to store the path on the class right now.

Reply:
We do not use right now, but I see no harm in having 'Config.path' either. It is just a matter that one of the two pull requests needs to make the change, if you want I can update my change and put the parameter on the constructor.